dashboards-observability/public/components/application_analytics/components/flyout_components/trace_detail_flyout.tsx

```diff
@@ -71,7 +72,13 @@ export function TraceDetailFlyout(props: TraceFlyoutProps) {
         <ServiceBreakdownPanel data={serviceBreakdownData} />
         <EuiSpacer size="xs" />
         <EuiHorizontalRule margin="s" />
-        <SpanDetailPanel traceId={traceId} http={http} colorMap={colorMap} />
+        <SpanDetailPanel
+          traceId={traceId}
+          http={http}
+          colorMap={colorMap}
+          page="app"
```

Comment:
np: in `create.tsx` we are using double quotes and brackets for the page parameter, can we maintain consistency with how we define the value?

Reply:
Yea that's a good point, I think most places it's just double quotes, I think I'll change them all to that.